packages/terra-clinical-item-view/src/terra-dev-site/doc/example/ItemViewTwoColumnByRow.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import IconBriefcase from 'terra-icon/lib/icon/IconBriefcase';
+import IconPerson from 'terra-icon/lib/icon/IconPerson';
```

Comment:
We can update these to use the new named exports introduced in https://github.com/cerner/terra-core/pull/3915.

Reply:
https://github.com/cerner/terra-clinical/pull/912/commits/abe1feec60bf2b34abcc990db0c2964416740638#diff-8a154576b72d3f91339b6cfa473a6c31078e664975dd661f3f49febe659a5ba4R2